lib/minidoc/timestamps.rb

```diff
@@ -15,6 +15,28 @@ def timestamps!
     end
   end
 
+  def set(attributes)
+    if self.class.record_timestamps
```

Comment:
I had the same confusion several times. Because we rely on module-inclusion for this feature, I actually think it might be reasonable to make including the module turn this on & call `timestamps!` itself. That would certainly simplify this module's code a good deal & avoid all the conditionals. That is, of course, a major version change.